File: nom-api/Nom.Api/Models/Person/PersonCreateResponseModel.cs
// Nom.Api/Models/Person/PersonCreateResponseModel.cs
namespace Nom.Api.Models.Person
{
    /// <summary>
    /// Represents the data of a Person profile returned by the API after a creation operation.
    /// </summary>
    public class PersonCreateResponseModel // UPDATED CLASS NAME
    {
        public long Id { get; set; }
        public required string Name { get; set; }
        public string? UserId { get; set; } // The linked IdentityUser ID
    }
}

File: nom-ui/src/app/person/models/person-create-response.model.ts
// File: nom-ui/src/app/person/models/person-create-response.model.ts

/**
 * Defines the structure of the response received from the backend
 * after a person record has been successfully created or updated.
 */
export interface PersonCreateResponseModel {
  id: number;
  name: string;
  userId: string | null;
}
